crates/frontend/src/pages/default_config/default_config.rs

```diff
@@ -297,6 +300,8 @@ pub fn default_config() -> impl IntoView {
                                             on:click=move |_| {
                                                 folder_click_handler(None);
                                                 enable_grouping.set(!enable_grouping.get());
+                                                set_local_storage("enable_grouping", &enable_grouping.get().to_string());
+                                                logging::log!("<<>> set this {:?}", &enable_grouping.get().to_string());
```

Comment:
if we need to log this can we change it to something better, or lets remove it if the log is not intended